### PR TITLE
możliwość sprawdzenia czy pracownik jest biedakiem

### PR DESCRIPTION
**Pull request by GdanskaProgramistka**

### CODE REVIEW

File: src/main/java/pakiet/Pracownik.java [hunk 1]
```diff
@@ -33,4 +33,6 @@ public boolean jestBogaczem() {
     public String duzeImie() {
         return imie.toUpperCase();
     }
+
+    public boolean jestBiedakiem() { return wyplata < 1000; }
```
Comment (boczkas) [inline src/main/java/pakiet/Pracownik.java]: Magic number - please create constant from it.

File: src/test/java/pakiet/PracownikTest.java [hunk 2]
```diff
@@ -54,6 +54,31 @@ public void jestBogaczem_wyplataPonizejProgu_zwracaFalse() {
         assertFalse(actual);
     }
 
+    @Test
```
Comment (boczkas) [inline src/test/java/pakiet/PracownikTest.java]: Should be parametrized test with multiple values.

File: src/test/java/pakiet/PracownikTest.java [hunk 3]
```diff
@@ -54,6 +54,31 @@ public void jestBogaczem_wyplataPonizejProgu_zwracaFalse() {
...
+        Pracownik typek = new Pracownik("Typkowski", 900);
+
+        //when
+        boolean actual = typek.jestBiedakiem();
+
+        //then
+        assertTrue(actual);
+    }
+
+    @Test
```
Comment (boczkas) [inline src/test/java/pakiet/PracownikTest.java]: Should be parametrized test with multiple values.

**Review by boczkas — CHANGES_REQUESTED** [review]
Please fix suggestions from comments

File: src/main/java/pakiet/Pracownik.java (hunk 1 above)
Comment (GdanskaProgramistka) [reply src/main/java/pakiet/Pracownik.java]: constant created

File: src/test/java/pakiet/PracownikTest.java (hunk 3 above)
Comment (GdanskaProgramistka) [reply src/test/java/pakiet/PracownikTest.java]: test parametrized added

**Review by boczkas — APPROVED** [review]
Approved

**Pull request merged**